### DISCUSSION TITLE
[MJX] ETA for muscle actuation

### DISCUSSION THREAD

**Discussion by driesmarzougui** [google-deepmind/mujoco#2183] [Asking for Help] [answered]
### Intro

Hi!

I am a PhD student at Ghent University, I use MuJoCo/MJX for my research on bio-inspired robotics.

### My question

In my current MJX project, I'm interested in comparing the resulting gaits (optimized via RL) between direct joint control and a model incorporating tendon and muscle dynamics. Do you have any updates on when muscle dynamics might be available in MJX?

### Confirmations

- [X] I searched the [latest documentation](https://mujoco.readthedocs.io/en/latest/overview.html) thoroughly before posting.
- [X] I searched previous [Issues](https://github.com/google-deepmind/mujoco/issues) and [Discussions](https://github.com/google-deepmind/mujoco/discussions), I am certain this has not been raised before.

**Comment by yuvaltassa** [comment]
This is currently WIP, should be out soon (1-2 weeks).
cc @thowell

**Reply by driesmarzougui** [reply]
Great! Thanks!

**Comment by thowell** [comment, marked as answer]
Muscle actuators have been added to MJX.
